README.md

```diff
@@ -2,4 +2,329 @@
 
 ## Objectives
 
-## Introduction
\ No newline at end of file
+1. Understand Swift collection types as generic collections.
+2. Recognize some implications of the pass-by-value nature of Swift collections.
+3. Create an explicitly-typed array.
+4. Use subscripting to interact with an array.
+5. Use a few array methods and properties to interact with an array.
+6. Create an explicitly-typed dictionary.
+7. Use subscripting to read a dictionary.
+8. Use subscripting to overwrite or add values for a key in a dictionary.
+9. Use the key-value Tuple returned by iterating over a dictionary with a `for-in` loop.
+
+## Collections
+
+Apple's summary of the conceptual differences between Swift's three collection types is quite perfect:
+
+>Swift provides three primary *collection* types, known as arrays, sets, and dictionaries, for storing collections of values. Arrays are ordered collections of values. Sets are unordered collections of unique values. Dictionaries are unordered collections of key-value associations.  
+>—From [*The Swift Programming Language, Collection Types*](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html#//apple_ref/doc/uid/TP40014097-CH8-ID105), by Apple, Inc.
+
+Arrays are ordered by index; sets hold unique values; dictionaries hold unique keys with values.
+
+**Objective-C:** *Swift's* `Array`, `Set`, *and* `Dictionary` *classes are interoperable with* `NSArray`, `NSSet`, *and* `NSDictionary`.
+
+### Generic Collections
+
+Since Swift is a strongly-typed language, its collection types are also "typed", meaning that they are given knowledge regarding their contents. Apple notes in the [*Collection Types* chapter](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html#//apple_ref/doc/uid/TP40014097-CH8-ID105) that its `Array`, `Set`, and `Dictionary` types "are implemented as *generic collections*."
+
+Uses of the keywords `T`, `Element`, and `Item` denote a *generic*. There is no meaningful distinction between the terms and you will see all three of them in various documents, both from Apple and from answers and blogs written by outside developers.
+
+We'll discuss generics in further detail in upcoming lessons. The effect of this is that the compiler gives individual instances of these collections knowledge of their membership types when they are declared, but otherwise the collection types themselves work exactly the same way whether they are told that they hold `Int`s, or `String`s, or anything else.
```

Comment:
"membership types" is a difficult phrase. Maybe going for something like "the effect of this is that the compiler knows what types (`String`, `Int`, etc...) is in the collection. Otherwise, the collection itself is completely identical".